util/pace/util/grid/generation.py

```diff
@@ -1529,6 +1560,64 @@ def rdyc(self) -> util.Quantity:
             gt4py_backend=self.dyc.gt4py_backend,
         )
 
+    def _init_cartesian(self):
+
+        domain_rad = PI / 16.0
+        lat_rad = self._deglat * PI / 180.0
+        lon_rad = 0.0
+
+        self._dx, self._dy = self._compute_dxdy_cartesian()
+        self._dx_agrid, self._dy_agrid = self._compute_dxdy_agrid_cartesian()
+        self._dx_center, self._dy_center = self._compute_dxdy_center_cartesian()
```

Comment:
Since dx, dxc, and dxa are all the same on an orthogonal grid I think you could consolidate these into one `_set_dxdy_cartesian` method

Reply:
I have three separate methods because `_dx`, `_dx_agrid`, and `_dx_center` (for example) sit at different locations on the grid. Could still work on reducing code duplication if you think that's worthwhile, e.g. with a single method like
```python
def _compute_dxdy_cartesian(self, dx_dims, dy_dims):
    dx_64 = self.quantity_factory.zeros(dx_dims, ...)
```
that can handle all three cases.